### ISSUE TITLE
Failed import from tree.taiga.io to private environment (invalid issues)

### ISSUE THREAD

**Issue by ijsf** [taigaio/taiga-back#713]
Today, I attempted to do a migration from our current tree.taiga.io environment to our own privately hosted Taiga environment running the latest stable version of taiga-back and taiga-front.

So during the import (e.g. after setting up a clean stable Taiga environment and using the Import function with a .json file supplied by the tree.taiga.io async exporter), taiga-back returned a `400 Bad Request`, which led me to try the command-line instead, hoping for better error reporting:

    python manage.py load_dump ~/bulk.json email@domain.tld

This threw the following, very non-descriptive and practically useless errors:

    Trying import local.py settings...
    ERROR: error importing issues
    {'issues': [{'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}, {'priority': ['This field is required.']}]}

By adding `print` commands to the `store_issue` function I was able to determine which issues were at fault, and noticed that all of these fields had the `priority` key set to `"None"`, which is technically correct, as the bulk defines `None` as one of the possible priorities:

    "priorities": [{"name": "None", "order": 0, "color": "#999999"}, {"name": "Low", "order": 1, "color": "#729fcf"}, {"name": "Normal", "order": 2, "color": "#4e9a06"}, {"name": "High", "order": 3, "color": "#CC0000"}],

By forcefully changing `"None"` to `"Normal"` in each of these issues, the import was successful and the new Taiga environment appeared to work.

Bottom line, the current stable importer is currently broken and is preventing us, and possibly others, from efficiently moving to our private environment.

**Comment by ijsf** [comment]
Updated issue according to new information.

**Comment by superalex** [comment]
Hello @ijsf 

I think this issue is related to the serialization of the "None" string, None in python means null (but null is not allowed for that attribute), when importing your json file taiga-back thinks that "None" string really means a python None. As a workaround you can just rename that priority to something different in the admin area of your project. Anyway we will try to solve it as soon as possible.

Regards,

**Comment by superalex** [comment]
This PR should fix the issue, I hope we can merge to stable and deploy it soon.

Regards,
